var/spack/repos/builtin/packages/atlas/package.py

```diff
@@ -107,28 +108,38 @@ def install(self, spec, prefix):
             make("install")
             self.install_test()
 
-    def setup_dependent_package(self, module, dspec):
-        # libsatlas.[so,dylib,dll ] contains all serial APIs (serial lapack,
-        # serial BLAS), and all ATLAS symbols needed to support them. Whereas
-        # libtatlas.[so,dylib,dll ] is parallel (multithreaded) version.
```

Comment:
I would keep this comment in your blas_libs method. I was wondering the same thing when I saw how many libraries it came with.